src/Test/L0/Worker/Build/TrackingManagerL0.cs

```diff
@@ -520,5 +520,12 @@ public void UpdatesTrackingConfigJobRunProperties()
                 Assert.True(DateTimeOffset.Now.AddSeconds(1) >= config.LastRunOn);
             }
         }
+
+        //removes CR so the test can run on Linux, even if it was synced on a Windows drive share
+        //this is used when Linux VM is building from a folder mounted on a Windows box
+        private string TrimCR(string value)
```

Comment:
"Remove" is more accurate than "Trim".

I don't really like this change, although I'm OK with it makes things easier for your development/testing. From my perspective, the issue really doesn't lie within the source code, and I don't like these kinds of workarounds littering the source code. The root issue is that you are building *from* the same network share (from multiple platforms). I'm surprised you haven't run into other issues by building off of the network share (e.g. intermediate build files refering to Windows style pathing). An alternative to this PR is to push your feature branch to the server and then pull from your other box.